app/assets/stylesheets/partials/wice_grid.scss

```diff
@@ -37,67 +32,61 @@
     }
 
     .collapse-multi-select-icon{
-      background: transparent url(/assets/icons/grid/collapse.gif) no-repeat;
+      background: transparent asset-url('/assets/icons/grid/collapse.gif') no-repeat;
     }
 
     .expand-multi-select-icon{
-      background: transparent url(/assets/icons/grid/expand.gif) no-repeat;
+      background: transparent asset-url('/assets/icons/grid/expand.gif') no-repeat;
     }
-
   }
 }
 
 .wice-grid{
-
-
   .desc,  .asc{
     padding-right: 18px;
     text-decoration:none;
   }
 
   .desc {
-    background: transparent url(/assets/icons/grid/arrow_down.gif) right no-repeat
+    background: transparent asset-url('/assets/icons/grid/arrow_down.gif') right no-repeat
   }
   .asc  {
-    background: transparent url(/assets/icons/grid/arrow_up.gif)   right no-repeat
+    background: transparent asset-url('/assets/icons/grid/arrow_up.gif')   right no-repeat
   }
 
   .submit.clickable{
-    background: transparent url(/assets/icons/grid/table_refresh.png) no-repeat;
+    background: transparent asset-url('/assets/icons/grid/table_refresh.png') no-repeat;
     @include icon-dimensions;
   }
 
   .reset.clickable{
-    background: transparent url(/assets/icons/grid/table.png) no-repeat;
+    background: transparent asset-url('/assets/icons/grid/table.png') no-repeat;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Prefer double-quoted strings